src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-pycountry.py

```diff
@@ -10,9 +10,9 @@
 # SPDX-License-Identifier: GPL-2.0-or-later
 # ------------------------------------------------------------------
 
-from PyInstaller.utils.hooks import collect_data_files
+from PyInstaller.utils.hooks import collect_data_files, copy_metadata
 
 # pycountry requires the ISO databases for country data.
 # Tested v1.15 on Linux/Ubuntu.
 # https://pypi.python.org/pypi/pycountry
-datas = collect_data_files('pycountry')
+datas = copy_metadata('pycountry') + collect_data_files('pycountry')
```

Comment:
Ahh, that solves [my confusion here](http://pyinstaller.47505.x6.nabble.com/PyInstaller-not-working-with-module-pycountry-tp3064p3095.html) on why the missing data files didn't show up sooner. Because we already had them here but adding another hook overwrites this one.